src/sqlparser/src/ast/mod.rs

```diff
@@ -1155,13 +1137,11 @@ impl fmt::Display for Statement {
                 name,
                 columns,
                 constraints,
-                table_properties,
                 with_options,
```

Comment:
Oops. How does the parser distinguish them?

Reply:
```rust
        let with_options = self.parse_with_properties()?;
        let table_properties = self.parse_options(Keyword::TBLPROPERTIES)?;
```